### DISCUSSION TITLE
using a capture card in linux

### DISCUSSION THREAD

**Discussion by whakru** [sblantipodi/firefly_luciferin#128] [Q&A] [answered]
is it possible to use a usb capture card for linux?  there is an idea to use hdmi -> usb card to capture video signal.  Based on orange pi 3

**Comment by sblantipodi** [comment, marked as answer]
hi @whakru,
welcome back!

Luciferin is focused on PC only currently and there is no plan to add an capture support at the moment.

**Reply by whakru** [reply]
I noticed that the ability to capture video via hdmi has become much easier.  Which immediately leads to the idea of ​​​​using raspberry / orange pi and others for an inexpensive backlight implementation.  I am currently using android tv with a closed platform.  and I suffer a lot from the lack of backlight) and immediately the idea came to use it through a capture card.  It's a pity, I hope it will be in the future) I'm sure that it will open a new level!

**Comment by sblantipodi** [comment]
I agree that the feature would be really cool, 
I haven't said that I'll never implement it but it's not in the todo list currently.

Luciferin is an open source project, if someone wants to join and contribute is really welcomed :)
